spark/src/test/java/org/opensearch/sql/spark/asyncquery/AsyncQueryExecutorServiceImplTest.java

```diff
@@ -44,6 +44,9 @@
 @ExtendWith(MockitoExtension.class)
 public class AsyncQueryExecutorServiceImplTest {
 
+  public static final String APPLICATION_ID = "00fd775baqpu4g0p";
+  public static final String EMR_JOB_EXECUTION_ROLE_ARN =
+      "arn:aws:iam::270824043731:role/emr-job-execution-role";
```

Comment:
curious: can these be TestConstants.EMRS_APPLICATION_ID and EMRS_EXECUTION_ROLE?

Reply:
Good point. Fixed.